lib/wibox/layout/ratio.lua

```diff
@@ -0,0 +1,342 @@
+---------------------------------------------------------------------------
+--- Fill all the available space. Each widget is assigned a ratio (percentage)
```

Comment:
This should probably have some prefix about being a layout - that might be missing in the others as well - but still.